src/components/taskDetails/index.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { ChangeEvent, FC } from 'react';
+import NavBar from '@/components/navBar/index';
+import TaskContainer from './TaskContainer';
+import Details from './Details';
+import classNames from './task-details.module.scss';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import task from '@/interfaces/task.type';
+import { useState, useEffect, useContext } from 'react';
+import useFetch from '@/hooks/useFetch';
+import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
+import fetch from '@/helperFunctions/fetch';
+
+type Props = {
+  url: string;
+  taskID: string;
+};
+
+const TaskDetails: FC<Props> = ({ url, taskID }) => {
+  const isAuthorized = useContext(isUserAuthorizedContext);
+  const [isEditing, setIsEditing] = useState(false);
+  const [initialData, setInitialData] = useState<task>();
+  const [taskDetails, setTaskDetails] = useState<task>();
+  const [editedDetails, setEditedDetials] = useState({});
+  const { response, error, isLoading } = useFetch(url);
+  const { SUCCESS, ERROR } = ToastTypes;
+
+  useEffect(() => {
+    const fetchedData: task = { ...response.taskData };
+    setTaskDetails(fetchedData);
+    setInitialData(fetchedData);
+  }, [isLoading, response]);
+
+  function convertTimeStamp(timeStamp: string) {
+    const dateTime = new Date(parseInt(timeStamp));
+    return dateTime.toLocaleString();
```

Comment:
we arent using this timestamp. We are supposed to use a standard timestamp.

Reply:
I'm just using whatever data the API returns.

Reply:
Updated with a standard Timestamp ✅